docker/docker-compose.yml

```diff
@@ -10,7 +10,8 @@ services:
       - POSTGRES_USER=puppetdb
       - POSTGRES_DB=puppetdb
     healthcheck:
-      test: ["CMD-SHELL", "pg_isready --username=puppetdb --dbname=puppetdb"]
+      # existence check for puppetdb database
+      test: [ 'CMD-SHELL', 'psql --username=puppetdb puppetdb -c ''''' ]
```

Comment:
Haha what is going on here with `''''` ?

Reply:
YAML requires escaping single quotes inside single quotes.

Let me see if I can wrap with double quotes and get rid of that to make it easier to read.

Reply:
Yeah, that works.. updated.

Reply:
What is actually going on with this call? What does `-c ""` do?

Reply:
It's an empty command (query) that just checks to make sure the database can be connected to by the given user

Reply:
Oh is this just testing if a database named `puppetdb` is created?

Similar to the `pg_isready` call but achieving it a different way?

Reply:
Ah okay.

What does the `pg_isready` do then, that didn't for us?

Reply:
I see your PR description answers my question 😄